server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1225,6 +1229,18 @@
             <element name="ws.inactivity.timeout" maxOccurs="1" minOccurs="0" type="gateway:TimeIntervalString">
                 <annotation/>
             </element>
+            <element name="http.keepalive" maxOccurs="1" minOccurs="0">
```

Comment:
Updated schema with docmentation for http.keepalive, http.keepalive.timeout. Will check with doc team.